plugins/woocommerce/tests/performance/requests/merchant/orders-filter.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-shadow */
+/**
+ * External dependencies
+ */
+import { sleep, check, group } from 'k6';
+import http from 'k6/http';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+	base_url,
+	cot_status,
+	admin_orders_base_url,
+	cot_admin_orders_base_url,
+	think_time_min,
+	think_time_max,
+	customer_user_id,
+} from '../../config.js';
+import {
+	htmlRequestHeader,
+	commonRequestHeaders,
+	commonGetRequestHeaders,
+	commonNonStandardHeaders,
+} from '../../headers.js';
+
+// Change URL if COT is enabled and being used
+let admin_orders_base;
+if ( cot_status === true ) {
+	admin_orders_base = cot_admin_orders_base_url;
+} else {
+	admin_orders_base = `${ admin_orders_base_url }&post_status=all`;
+}
+
+export function ordersFilter() {
+	let response;
+
+	const date = new Date();
+	const month = date.toJSON().slice( 5, 7 );
```

Comment:
Would be simpler just to do something like `const month = date.getMonth();`?

https://www.w3schools.com/jsref/jsref_getmonth.asp

If that works, we could do the same with `year`.

Reply:
unfortunately it has to match the format of the request which needs a leading zero

Reply:
I've been checking the Date docs and it seems it is poor in terms of format. I found other approaches to do it using the objects it provides, but it wouldn't add any considerable advantage to the current approach since you would still need to transform the data and it wouldn't read easily.